[2022-06-18 06:52:36,673] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: coffee_lake_demo-v2.load_data manual__2022-06-18T06:52:30.500581+00:00 [queued]>
[2022-06-18 06:52:36,689] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: coffee_lake_demo-v2.load_data manual__2022-06-18T06:52:30.500581+00:00 [queued]>
[2022-06-18 06:52:36,689] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-18 06:52:36,690] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-18 06:52:36,690] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-18 06:52:36,714] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): load_data> on 2022-06-18 06:52:30.500581+00:00
[2022-06-18 06:52:36,722] {standard_task_runner.py:52} INFO - Started process 235 to run task
[2022-06-18 06:52:36,727] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'coffee_lake_demo-v2', 'load_data', 'manual__2022-06-18T06:52:30.500581+00:00', '--job-id', '492', '--raw', '--subdir', 'DAGS_FOLDER/scraper_v2_dag.py', '--cfg-path', '/tmp/tmpa63l459f', '--error-file', '/tmp/tmp3nsmskyq']
[2022-06-18 06:52:36,729] {standard_task_runner.py:80} INFO - Job 492: Subtask load_data
[2022-06-18 06:52:36,811] {task_command.py:370} INFO - Running <TaskInstance: coffee_lake_demo-v2.load_data manual__2022-06-18T06:52:30.500581+00:00 [running]> on host 312614bad459
[2022-06-18 06:52:36,937] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=coffee_lake_demo-v2
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-06-18T06:52:30.500581+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-18T06:52:30.500581+00:00
[2022-06-18 06:53:05,949] {python.py:173} INFO - Done. Returned value was: None
[2022-06-18 06:53:05,960] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=coffee_lake_demo-v2, task_id=load_data, execution_date=20220618T065230, start_date=20220618T065236, end_date=20220618T065305
[2022-06-18 06:53:06,001] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-18 06:53:06,026] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
